src/inject/dynamic-theme/index.ts

```diff
@@ -25,50 +26,52 @@ let filter: FilterConfig = null;
 let fixes: DynamicThemeFix = null;
 let isIFrame: boolean = null;
 
-function createOrUpdateStyle(className: string, root: ParentNode = document.head || document) {
-    let style = root.querySelector(`.${className}`) as HTMLStyleElement;
-    if (!style) {
-        style = document.createElement('style');
-        style.classList.add('darkreader');
-        style.classList.add(className);
-        style.media = 'screen';
+function createOrUpdateElement(className: string, root: ParentNode = document.head || document, elementType = 'style') {
```

Comment:
Let's create 2 functions: one for style and one for script, because it is not obvious that calling `createOrUpdateElement(className)` returns a style element.